README.md

```diff
@@ -21,35 +21,175 @@ The testing tasks are based on [istio.io Doc Tasks](https://istio.io/v1.9/docs/t
+
+For example, to run the `TestFaultInjection` test case, run the following command:
+```console
+make test TestFaultInjection
+```
+
+Alternatively, you can run a specific test case by specifying the name in the `TEST_CASE` environment.
+
+For example, to run the `TestFaultInjection` test case, run the following command:
+```console
+TEST_CASE=TestFaultInjection make test
+```
+
+
+See the `*_test.go` source files to find the test case names.
+
+
+### Running against a single ServiceMeshControlPlane version
+
+By default, `make test` runs test cases against all supported versions of `ServiceMeshControlPlane`. When you want to run against a single `ServiceMeshControlPlane` version, specify the version in the `SMCP_VERSION` environment variable.
+
+For example, to run the tests against version `v2.4`, run them as follows:
+```console
+SMCP_VERSION=2.4 make test
+```
+
+NOTE: you may include or omit the `v` prefix in the version number. 
+
+
+### Running on architectures other than x86
+
+By default, the tests assume that the cluster nodes use the x86 architecture. If your cluster uses a different architecture, set the `OCP_ARCH` environment variable before running the tests.
+
+For IBM Power Systems, run tests with: 
+```
+OCP_ARCH=p make test
+```
+
+For IBM zSystems, run tests with:
+```
+OCP_ARCH=z make test
+```
+
+For ARM-based clusters, run tests with: 
+```
+OCP_ARCH=arm make test
+```
+
+
+### Running on Red Hat Openshift Service on AWS (ROSA)
+
+To run tests on Red Hat Openshift Service on AWS (ROSA), set the `ROSA` environment variable to `true`:
+
+```console
+ROSA=true make test
+```
+
+### Running multi-cluster test cases
+
+The test suite contains both single- and multi-cluster test cases. 
+By default, only single-cluster test cases are run. 
+To run multi-cluster tests, ensure that the `KUBECONFIG2` environment variable points to the `kubeconfig` file for the second cluster. 
+
+
+### Running tests in VSCode, GoLand, or another IDE
+
+The tests in maistra-test-tool are standard go tests and can be run in an IDE using standard methods. No special setup is required.
+
+### Reducing log output
+
+Due to the eventually-consistent nature of OpenShift clusters, each test performs a series of retry attempts of each action it performs. 
+By default, each attempt is logged, which typically results in a series of transient failures being shown in the log. 
+You can prevent maistra-test-tool from logging each attempt by setting the `LOG_FAILED_RETRY_ATTEMPTS` environment variable to `false`.
+The verbose output is mostly useful while writing or debugging tests. In CI systems, using the less-verbose form might be preffered.
+
+To disable verbose logging, run the tests as follows:
+```console
+LOG_FAILED_RETRY_ATTEMPTS=false make test
+```
+
+### Running tests in a container
+
+You can also run the test suite in a container, using the image `quay.io/maistra/maistra-test-tool:2.4`. 
```

Comment:
Do we want to use the 'latest' tag as it's defined here https://github.com/maistra/maistra-test-tool/pull/514/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L11 ?